cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -1586,7 +1586,7 @@ private TSStatus processPartitionedPlan(PhysicalPlan plan) throws UnsupportedPla
           }
           return executeNonQuery(plan);
         } catch (MetadataException e) {
-          logger.error("Failed to set storage group or create timeseries, because {}", e);
+          logger.error(String.format("Failed to set storage group or create timeseries, because %s", e));
```

Comment:
Why change this?

Reply:
It was recognized as a bug according to rules of sonar, "Fewer arguments provided (0) than placeholders specified (1) ".